jigglypuffRL/common/utils.py

```diff
@@ -0,0 +1,26 @@
+import torch
+
+
+def save_params(algo, name=None, version=None):
+    if name is None:
+        name = algo.network_type
+
+    checkpoint = {'algo': name,
+                  'weights': algo.ac.state_dict()}

```

Comment:
There are other params that need to be saved as well. Especially since its RL, other params are very important e.g. gamma, tau, actor and critic LRs. We don't have a lot of these yet in the class object but we plan to add them in the future. So can you change this to something like:
```
checkpoint = self.__dict__
checkpoint['algo'] = name
checkpoint['weights'] = algo.ac.state_dict()
```

Reply:
Generalized it to save all inputted parameters. Will directly change the checkpoint dict.